src/inspect_ai/log/_file.py

```diff
@@ -95,6 +101,45 @@ def write_eval_log(log: EvalLog, log_file: str | FileInfo) -> None:
         f.write(eval_log_json(log))
 
 
+def write_log_dir_manifest(
+    log_dir: str,
+    *,
+    filename: str = "manifest.json",
+    output_dir: str | None = None,
+    fs_options: dict[str, Any] = {},
+) -> None:
+    """Write a manifest for a log directory.
+
+    A log directory manifest is a dictionary of EvalLog headers (EvalLog w/o samples)
+    keyed by log file names (names are relative to the log directory)
+
+    Args:
+      log_dir (str): Log directory to write manifest for.
+      filename (str): Manifest filename (defaults to "manifest.json")
+      output_dir (str | None): Output directory for manifest (defaults to log_dir)
+      fs_options (dict[str,Any]): Optional. Additional arguments to pass through
+        to the filesystem provider (e.g. `S3FileSystem`). Use `{"anon": True }`
+        if you are accessing a public S3 bucket with no credentials.
```

Comment:
Seems unlikely you'd be uploading to a public S3 bucket with no credentials

Reply:
Agreed. I removed that bit of the doc comment (but kept `fs_options` as with more exotic filesystems/configs it can sometimes be necessary to pass custom params through)